assets/component-slider.css

```diff
@@ -118,15 +118,37 @@ slider-component {
   display: block;
 }
 
-.slider-counter__link--active.slider-counter__link--dots .dot{
+.slider-counter__link--active.slider-counter__link--dots .dot {
   background-color: rgb(var(--color-foreground));
 }
 
+.slider-counter__link--dots:not(.slider-counter__link--active):hover .dot {
+  background-color: rgba(var(--color-foreground), 0.3);
+  border: 0;
+}
+
+.slider-counter__link--dots .dot,
+.slider-counter__link--numbers {
+  transition: transform 0.2s ease-in-out;
+}
+
+.slider-counter__link--active.slider-counter__link--dots .dot,
+.slider-counter__link--active.slider-counter__link--numbers,
+.slider-counter__link--dots:hover .dot,
+.slider-counter__link--numbers:hover {
+  transform: scale(1.4);
+}
```

Comment:
@melissaperreault here I played with the hover state for the sake of exploration. The test store link has been updated in the PR and here is a video recap: [video](https://screenshot.click/21-11-z06vj-m00hl.mp4)

Reply:
I like your suggestion to scale, but I think we can reduce it slightly.

**For the numbers:** [Video example](https://screenshot.click/21-53-t3x8y-wyic4.mp4)
- [ ] Can we change `transform: scale(1.4);` to `transform: scale(1.1);`
- [ ] Also remove the underline on hover? Scale and opacity should be enough and this way the underline is reserved for the active state. 

**For the dots:** We may need to pair on this one to get the exact values, but can we try this?
- [ ] Can the active dots remain the same size as the other dots? [Screenshot](https://screenshot.click/21-11-ezqsq-yp1g4.png)
- [ ] On hover can we change `transform: scale(1.4);` to `transform: scale(1.1);`
- [ ] I can't figure out where the values are in the code, but on hover can we try to increase the stroke of the outline instead of filling in the circle? (This may not look great, but just wanted to try) 

**Arrows**
- [ ] What is the scaling added here? Can we try the same scaling used on the dots and numbers?

**Pagination**
- [ ] Can we update the pagination to match the styling for numbers 


cc @ludoboludo @melissaperreault

Reply:
Should be good now 👍  If you want to check the [linked demo store in the PR description](https://os2-demo.myshopify.com/admin/themes/126938021910/editor)

Reply:
The active dot is still slightly larger than the non-active dots. [Screenshot.](https://screenshot.click/21-30-wvg5l-0a818.png) Otherwise, it's perfect 👍

Reply:
Sweet, done 👌